uptane-standard.md

```diff
@@ -780,15 +790,19 @@ The ECU SHALL verify that the latest image matches the latest metadata as follow
 
 If the ECU has secondary storage, the checks SHOULD be performed on the image in secondary storage, before it is installed.
 
-If any step fails, the ECU SHALL jump to the fifth step ({{create_version_report}}). If a step fails and the ECU does not have secondary storage, and the ECU has created a backup of its previous working image, the ECU SHOULD now install the backup image.
+If any step fails, the ECU SHALL jump to the fifth step ({{create_version_report}}). If a step fails and the ECU does not have secondary storage, then the ECU SHALL ensure a backup of a previous working image is stored before attempting the verification. Upon failure, the ECU SHALL install the backup image to ensure it is in a working condition.
```

Comment:
```suggestion
If any step fails, the ECU SHALL jump to the ({{create_version_report}}) step. If a step fails and the ECU does not have secondary storage, then the ECU SHOULD ensure a backup of a previous working image is stored before attempting the verification. Upon failure, the ECU SHOULD install the backup image to ensure it is in a working condition.
```

Reply:
We cannot force secondary storage on all implementations, therefore the SHALL -> SHOULD

Reply:
I agree with not forcing all ECUs to have secondary storage, sorry for my confusion.
My intent was to force implementor's to have a backup of current running image & associated metadata in case of update failure.
This backup can be stored on the updating ECU, on another ECU with sufficient storage, on a dealer tool, etc.

Does this clarify my intent?
I will provide some clarity in my wording...